Modules/SubModule_Import/xASL_imp_DCM2NII_Subject.m

```diff
@@ -207,8 +207,11 @@
                 end
                 
                 %% 4.8 In case of GE or PARREC/a single NII ASL file loaded from PAR/REC, we need to shuffle the dynamics from CCCC...LLLL order to CLCLCLCL... order
-                [nii_files, summary_line, globalCounts] = xASL_imp_DCM2NII_Subject_ShuffleTheDynamics(globalCounts, scanpath, scan_name, nii_files, iSubject, iSession, iScan);
                 
+                %Adding x struct to the next function will allow the recognition of Hadamard data using the studyPar (sometimes there's no info in the SeriesDescription)
```

Comment:
Don't know if we really need this comment, not that it's wrong, but then I'd rather write down what the x struct is used for in the header of xASL_imp_DCM2NII_Subject_ShuffleTheDynamics.